src/app/_components/news/index.jsx

```diff
@@ -0,0 +1,112 @@
+'use client'
+
+import { Swiper, SwiperSlide } from 'swiper/react'
+import { Navigation } from 'swiper/modules'
+import 'swiper/css'
+import 'swiper/css/navigation'
+import styles from './index.module.scss'
+
+import { MoreButton } from '@/components/MoreButton'
+
+const jobs = [
+  {
+    id: 1,
+    img: '/news/job1-sp.png',
+    width: 370,
+    height: 208,
+    title: '平日・夕方までの勤務!伊勢市の伊勢赤十字病院で医療事務求人',
+    salaryLabel: '月給',
+    salaryDetail: '月給 148,000円～',
+    placeLabel: '所在地',
+    placeDetail: '三重県伊勢市',
+    stationLabel: '最寄り駅',
```

Comment:
[must]
デザインと同様になるように修正をお願いします！

```suggestion
    stationLabel: '最寄駅',
```

Reply:
修正いたしました。
fea95fbe02a55b0b0f7120cb3a6e1cad1959489e